ocaml/parsing/parsetree.mli

```diff
@@ -513,52 +513,6 @@ and function_param_desc =
       - [?l:(P = E0)]
         when [lbl] is {{!Asttypes.arg_label.Optional}[Optional l]}
         and [exp0] is [Some E0]
-<<<<<<< HEAD
```

Comment:
I think there is a merge conflict farther up in this file that remains unresolved.